rules/linux/lateral_movement_telnet_network_activity_external.toml

```diff
@@ -20,20 +20,21 @@ false_positives = [
 ]
 from = "now-9m"
 index = ["auditbeat-*", "logs-endpoint.events.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License"
 name = "Connection to External Network via Telnet"
 risk_score = 47
 rule_id = "e19e64ee-130e-4c07-961f-8a339f0b8362"
 severity = "medium"
 tags = ["Elastic", "Host", "Linux", "Threat Detection", "Lateral Movement"]
 timeline_id = "76e52245-7519-4251-91ab-262fb1a1728c"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:network and event.type:(connection or start) and
-  process.name:telnet and
-  not destination.ip:(127.0.0.0/8 or 10.0.0.0/8 or 172.16.0.0/12 or 192.168.0.0/16 or "FE80::/10" or "::1/128")
+sequence by process.entity_id
+  [process where process.name : "telnet" and event.type == "start"]
```

Comment:
`process.name : "telnet"` could possibly be changed to `process.name == "telnet"` because Linux

Reply:
definitely! i think linux should use `==` when possible

Reply:
```suggestion
  [process where process.name == "telnet" and event.type == "start"]
```